src/main.js

```diff
@@ -1,19 +1,50 @@
 // Create variables targetting the relevant DOM elements here 👇
-
+var coverPic = document.querySelector('.cover-image')
+var coverTitle = document.querySelector('.cover-title')
+var coverTagLine1 = document.querySelector('.tagline-1')
+var coverTagLine2 = document.querySelector('.tagline-2')
+var priceTag = document.querySelector('.price-tag')
+var randomBtn = document.querySelector('.random-cover-button')
 
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
```

Comment:
I think the project wants us to use this "Cover" javascript class! It's defined in a different javascript file at 'src/Cover.js'. Check it out! Let's see if we can figure out how to refactor our code so that it instantiates a new object of the Cover class. If that sounds confusing, I recommend reviewing your notes/lesson on javascript Classes (https://frontend.turing.edu/lessons/module-1/js-classes.html) before we code so you remember the terms!